packages/app-desktop/gui/MainScreen.tsx

```diff
@@ -121,6 +122,18 @@ const defaultLayout: LayoutItem = {
 	],
 };
 
+const layoutKeyToLabel = (key: string, plugins: PluginStates) => {
+	if (key === 'sideBar') return _('Sidebar');
+	if (key === 'noteList') return _('Note list');
+	if (key === 'editor') return _('Editor');
+
+	const viewInfo = pluginUtils.viewInfoByViewId(plugins, key);
+	if (viewInfo) {
+		return PluginService.instance().safePluginNameById(viewInfo.plugin.id);
```

Comment:
```suggestion
		return PluginService.instance().pluginNameById(viewInfo.plugin.id);
```
The above is currently used to label panels from plugins. The "safe" in "safePluginNameById" was intended to mean that the method doesn't `throw` if the plugin isn't installed/loaded. However, just `pluginNameById` may make more sense.